app/src/main/java/com/example/myweatherapp/utils/WeatherUtils.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.myweatherapp.utils
+
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+const val CITY = "Tandil"
+const val EMPTY_STRING = ""
+const val URL = "http://openweathermap.org/img/wn/"
+const val FORMAT = "@2x.png"
+val formatJson = SimpleDateFormat("yyyy-MM-dd HH:mm:ss", Locale.ENGLISH)
```

Comment:
keep the SimpleDateFormat(...) in the method in which you use it and the "yyyy-MM-dd HH:mm:ss" here

Reply:
Done thx